components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/ClinicalStatusController.java

```diff
@@ -90,6 +91,10 @@ public String getName()
     @Override
     public void writeJSON(Patient patient, JSONObject json, Collection<String> selectedFieldNames)
     {
+        if (selectedFieldNames != null && !selectedFieldNames.contains(UNAFFECTED)) {
```

Comment:
Ideally, this should check if ```selectedFieldNames``` contains ```getName()```, but the export UI doesn't work that way.